.drone.yml

```diff
@@ -1,120 +1,149 @@
 ---
 kind: pipeline
 type: kubernetes
-name: deploy
+name: deploy tag
+trigger:
+  event:
+    - tag
+  branch:
+    - main
+
+steps:
+  - name: checkout
+    image: quay.io/ukhomeofficedigital/hocs-helm-kubectl:1.0.0
+    commands:
+      - git fetch --tags
+      - git checkout ${DRONE_TAG}
+      - git submodule update --init --recursive
+
+  - &deploy
+    name: cs-dev
+    image: quay.io/ukhomeofficedigital/hocs-helm-kubectl:1.0.0
+    environment:
+      CHART_NAME: hocs-docs-converter
+      KUBE_CLUSTER: acp-notprod
+      KUBE_NAMESPACE: cs-dev
+      KUBE_SERVER: https://kube-api-notprod.notprod.acp.homeoffice.gov.uk
+      KUBE_TOKEN:
+        from_secret: cs_notprod
+      VERSION: ${DRONE_TAG}
+    commands:
+      - ./ci/helm/helm-deploy.sh --values ./helm/values-notprod.yaml --values ./helm/values-dev.yaml
+    depends_on:
+      - checkout
+
+  - <<: *deploy
+    name: wcs-dev
+    environment:
+      KUBE_NAMESPACE: wcs-dev
+    depends_on:
+      - cs-dev
+---
+kind: pipeline
+type: kubernetes
+name: promote
 trigger:
   event:
     - promote
 
+environment:
+  CHART_NAME: hocs-docs-converter
+  KUBE_CLUSTER: acp-notprod
+  KUBE_SERVER: https://kube-api-notprod.notprod.acp.homeoffice.gov.uk
+
 steps:
-  - name: fetch and checkout
+  - name: checkout
     image: alpine/git
     commands:
       - git fetch --tags
-      - git checkout $${VERSION}
+      - git checkout ${VERSION}
+      - git submodule update --init --recursive
 
-  - name: deploy to cs-qa
-    image: quay.io/ukhomeofficedigital/kd
-    commands:
-      - cd kube
-      - ./deploy.sh
+  - name: cs-qa
+    image: quay.io/ukhomeofficedigital/hocs-helm-kubectl:1.0.0
     environment:
-      ENVIRONMENT: cs-qa
+      KUBE_NAMESPACE: cs-qa
       KUBE_TOKEN:
-        from_secret: hocs_docs_converter_cs_qa
-      KUBE_SERVER: https://kube-api-notprod.notprod.acp.homeoffice.gov.uk
+        from_secret: cs_notprod
+    commands:
+      - ./ci/helm/helm-deploy.sh --values ./helm/values-notprod.yaml --values ./helm/values-qa.yaml
+    depends_on:
+      - checkout
     when:
       target:
-        - release
-    depends_on:
-      - fetch and checkout
+        - cs-qa
```

Comment:
It's a small issue and one that might be acceptable if we want to get rid of the `release` way. But this should also have a target of `release`.